Makefile

```diff
@@ -532,13 +532,13 @@ rabbitmq_deploy_prep: rabbitmq_deploy_cleanup ## prepares the CR to install the
 .PHONY: rabbitmq_deploy
 rabbitmq_deploy: input rabbitmq_deploy_prep ## installs the service instance using kustomize. Runs prep step in advance. Set RABBITMQ_REPO and RABBITMQ_BRANCH to deploy from a custom repo.
 	$(eval $(call vars,$@,rabbitmq))
-	#oc kustomize ${DEPLOY_DIR} | oc apply -f -
-	oc apply -f ${DEPLOY_DIR}/rabbitmq.yaml
+	KIND=RabbitmqCluster NAME=rabbitmq bash scripts/gen-name-kustomize.sh
```

Comment:
Not thrilled about requiring a new bash script just for this, but I understand you are doing it this way to leverage it in the future.